packages/block-library/src/gallery/gallery.js

```diff
@@ -51,45 +42,14 @@ export const Gallery = ( props ) => {
 				'is-cropped': imageCrop,
 			} ) }
 		>
-			<ul className="blocks-gallery-grid">
-				{ images.map( ( img, index ) => {
-					const ariaLabel = sprintf(
-						/* translators: 1: the order number of the image. 2: the total number of images. */
-						__( 'image %1$d of %2$d in gallery' ),
-						index + 1,
-						images.length
-					);
+			<InnerBlocks
+				allowedBlocks={ [ 'core/image' ] }
+				__experimentalTagName="ul"
+				__experimentalItemCallback={ ( item ) => (
+					<li className="wp-block-gallery__image">{ item }</li>
+				) }
+			></InnerBlocks>
```

Comment:
This should just use a self-closing tag.
```suggestion
			<InnerBlocks
				allowedBlocks={ [ 'core/image' ] }
				__experimentalTagName="ul"
				__experimentalItemCallback={ ( item ) => (
					<li className="wp-block-gallery__image">{ item }</li>
				) }
			/>
```

Reply:
Fixed